tests/Integration/Infrastructure/Controller/Map/Fragment/MapDataControllerTest.php

```diff
@@ -8,30 +8,35 @@
+        $this->assertSame('{"type":"FeatureCollection","features":[]}', $data);
     }
 
-    public function testGetFilters(): void
+    public function testMeasureTypesFilter(): void
     {
         $client = static::createClient();
-        $client->request('GET', '/carte/data.geojson?map_filter_form%5Bcategory%5D=permanents_only&map_filter_form%5B_token%5D=4ef8515d4d5e05fb9f.8Zo-nuv_uVZHlzVSYd0sF6Lgr2Sv0ZOEh2efPahYS6g.vP9H-bO00Wcm51QbUuUdfe6SwwnmptT00QLoBJ87CuTF7kTqparQYCLQRg');
+        $client->request('GET', '/carte/data.geojson?map_filter_form[measureTypes][]=noEntry');
 
         $this->assertResponseStatusCodeSame(200);
         $this->assertSecurityHeaders();
         $this->assertResponseHeaderSame('Content-Type', 'application/json');
 
-        $data = $client->getResponse()->getContent();
-        $this->assertSame('{"type":"FeatureCollection","features":[]}', $data);
+        $jsonData = $client->getResponse()->getContent();
+        $dataArray = json_decode($jsonData, true);
+
+        $measureType = [];
+        foreach ($dataArray['features'] as $feature) {
+            $measureType[] = $feature['properties']['measure_type'];
+        }
+        $this->assertContains('noEntry', $measureType);
```

Comment:
Attention, ce bout de code ne vérifie que la présence _d'au moins un noEntry_ dans `$measureTypes`

Ce que l'on voudrait faire c'est vérifier que _chaque feature_ concerne une mesure de type noEntry

Pour cela il faut d'abord vérifier que le nombre de features est celui attendu, ensuite on peut mettre des assertSame dans une boucle

(+) Utiliser l'enum `MeasureTypeEnum`

```suggestion
        $this->assertCount(3, $dataArray['features']); // 3 est un exemple, utiliser le vrai nombre attendu

        foreach ($dataArray['features'] as $feature) {
            $this->assertSame(MeasureTypeEnum::NO_ENTRY->value, $feature['properties']['measure_type']);
        }
```